ballerina/llm_client_azure_open_ai.bal

```diff
@@ -24,6 +24,11 @@ type AzureOpenAIModelConfig record {|
     string serviceUrl;
 |};
 
+type SchemaResponse record {
```

Comment:
This is share across both clients, right? If so, it needs to go in main.bal file.